.pylintrc

```diff
@@ -186,7 +186,7 @@ max-nested-blocks=5
 [FORMAT]
 
 # Maximum number of characters on a single line.
-max-line-length=100
+max-line-length=135
```

Comment:
Is there any way to comply with max-line-length=100? We could dry up the long lines that was annoying pylint.